boilerplate/app/navigators/root-navigator.tsx

```diff
@@ -29,6 +29,7 @@ const RootStack = () => {
   return (
     <Stack.Navigator
       screenOptions={{
+        cardStyle: { backgroundColor: "#5D2555" },
```

Comment:
Could we store this color in the palette?

https://github.com/infinitered/ignite/blob/eadde266f6b46659bac9372056face896d602b61/boilerplate/app/theme/palette.ts

Reply:
Yep. I considered doing that, but decided it was outside the scope of this PR as that would require changing it in the main template as well.

However, I'll happily add another commit to this same PR to fix both issues.

Reply:
I've just pushed these changes.

I also noticed that the palette file (and other hex colors throughout the app) have inconsistent casing. Would you like me to submit another PR to standardize them? If so, what's your preference - uppercase or lowercase?

Reply:
Thanks! This branch looks good to merge @jamonholmgren. 

I don't think we have a preference on the hex capitalization. people are likely to replace those with their own colors anyways. If you send a PR, we'd probably merge it though :D